frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -187,6 +186,18 @@ export const funnelLogic = kea<funnelLogicType>({
                         return EMPTY_FUNNEL_RESULTS
                     }
 
+                    // Don't load results if layout was the only thing changed
+                    if (
+                        equal(
+                            Object.assign({}, values.filters, { layout: undefined }),
```

Comment:
quick q: why do we need to assign anything to layout at all if we are only comparing filters? does values.filters contain a `layout` key as well?

Reply:
Yup, both `values.filters` and `values.lastAppliedFilters` contain layout keys (see param [here](https://github.com/PostHog/posthog/pull/5576/files/e9ab8fdb5b0dca33780174fe85cdd531e5743632#diff-92e32501a6b77e6bdff549cb4f721a309c1b5fd3df14615dc6969315b66041dcL112)). By assigning the layout to undefined, I'm normalizing layouts across both objects so that any layout differences here are canceled out.